src/commands/periscope/helpers/periscopehelper.ts

```diff
@@ -95,7 +97,11 @@ export async function getStorageInfo(
         const storageAccKeyList = await storageClient.storageAccounts.listKeys(resourceGroupName, accountName);
         const storageKey = storageAccKeyList.keys?.find((it) => it.keyName === "key1")?.value!;
 
+        // Get container name from cluster-info default behaviour was APIServerName without
+        const containerName = await extractContainerName(clusterKubeConfig!);
+
         const clusterStorageInfo = {
+            containerName: containerName!,
```

Comment:
Again, you can't just `!` this away without validating.  If `extractContainerName` is guaranteed to return a string, _change the signature._  If it's not, _validate before asserting._